webpack.config.js

```diff
@@ -208,6 +208,10 @@ if ( calypsoEnv === 'production' ) {
 	) );
 }
 
+if ( ! config.isEnabled( 'desktop' ) ) {
```

Comment:
This looks to be a reasonable solution! - we would never need to use this module in cases where desktop was enabled 👍

With this might be able to remove some of the checking lines from @gwwar's recent changes - what do you guys think?